scripts/helpers/utilities.mk

```diff
@@ -440,16 +443,23 @@ define prepare_loader_cache
         cat "$(DYNAMIC_LOAD)/sys32.ld.conf" >> "$(DYNAMIC_LOAD)/ld.so.conf"; \
     fi; \
     grep -E -v "^$$" "$(DYNAMIC_LOAD)/ld.so.conf" > "$(AT_DEST)/etc/ld.so.conf"; \
-    if [[ $1 -eq 2 ]]; then \
-        rm -rf $(AT_DEST)/syslib-override; \
-    else \
-        for cpu in $(BUILD_ACTIVE_MULTILIBS); do \
-            mkdir -p $(AT_DEST)/syslib-override/"$${cpu}"; \
-            (\cd $(AT_DEST)/lib64 && \
-             find . -maxdepth 1 -type f \
-                  -exec cp -p '{}' $(AT_DEST)/syslib-override/"$${cpu}"/'{}' \; \
-            ); \
-        done; \
+    if [[ "$(AT_MAJOR_VERSION)" > 15 ]]; then \
```

Comment:
Why is this code restricted to `AT > 15`?
What happens on AT 13, 14 and 15? Don't they need `syslib-override`?

Reply:
I confess to some confusion on my part here. When I did not so restrict, the builds failed. It may have been my sloppiness in cleaning up after faking the `syslib-override` resulting in files in the build tree which were not packaged, and the requisite errors.  That being said, the builds now all work.  The earlier AT releases may be benefiting from using `DT_RUNPATH` and not using the loader cache, though. The right thing to do is probably maintain the `syslib-override` behavior (likely renamed `glibc-hwcaps` for consistency with modern glibc).